modules/mocked_authentication/app/controllers/mocked_authentication/credential_controller.rb

```diff
@@ -29,6 +29,18 @@ def credential_list
       render json: { errors: e }, status: :bad_request
     end
 
+    def index
+      type = params[:type].presence
+
+      validate_index_params(type)
```

Comment:
In `validate_index_params`, can you add a check for the `state` param?

Also, copy the existing `validate_index_params` and rename it to `validate_credential_list_params` and have the `credential_list` function use that function instead

Reply:
I'll try my best 👍